ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -284,11 +282,11 @@ void RSkComponentImage::requestNetworkImageData(ImageSource source) {
   if(hasToTriggerEvent_) sharedCurlNetworking->sendRequest(curlRequest,query);
 }
 
-void RSkComponentImage::didSendEvent(bool didSuccess) {
+void RSkComponentImage::sendEvents(bool hasToSendLoadEvent) {
   auto component = getComponentData();
   auto imageEventEmitter = std::static_pointer_cast<ImageEventEmitter const>(component.eventEmitter);
 
-  if(didSuccess)
+  if(hasToSendLoadEvent)
     imageEventEmitter->onLoad();
```

Comment:
This can also be moved to sendEvents Api . With that redundancy on setting up hasToLoadEvent & retrieving Image EventEmitter can be avoided 

Ex: emitImageEvents(imageEvent event){ };